DAQ_System/Program/Src/app.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+* DataAcquisition2.0 — 2023 Vehicle
+* (c) 2022 Dallas Formula Racing - Embedded Firmware Team
+* Formula SAE International Collegiate Chapter
+* GPL-3.0 License
+*/
+
+
+// Standard Libraries
+#include <stdio.h>
+#include <inttypes.h>
+
+#include <memory>
+
+// ST HAL Dependencies
+#include "gpio.h"
+
+#include "usart.h"
+extern UART_HandleTypeDef huart3;
+
+#include "adc.h"
+extern ADC_HandleTypeDef hadc1;
+
+// DFR Custom Dependencies
+#include "app.hpp"
+#include "../../Core/Inc/retarget.h"
+#include "Sensor/LinearPotentiometer/ilinear_potentiometer.hpp"
+#include "Sensor/LinearPotentiometer/sls1322.hpp"
+
+
+void cppMain() {
+	// Enable `printf()` using USART
+	RetargetInit(&huart3);
```

Comment:
why no huart1

Reply:
The Nucleo-F429ZI uses USART3 to connect to ST-Link, instead of USART 1 or 2.